crates/viewer/re_ui/src/design_tokens.rs

```diff
@@ -33,6 +33,19 @@ impl AlertVisuals {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Default)]
+pub enum TableStyle {
+    /// Used for presenting a lot of information to the user
+    /// without wasting vertical space,
+    /// like when showing log output.
+    #[default]
+    Dense,
+
+    /// Used when we want more margins and spacing,
+    /// e.g. in the catalog view.
+    Spacious,
```

Comment:
maybe worth calling out that a prime motivation for using this is that there may be additional buttons etc.